doc/sphinx/practical-tools/utilities.rst

```diff
@@ -515,6 +515,11 @@ To build, say, two targets foo.vo and bar.vo in parallel one can use
   + ``-extra-phony`` and ``-extra`` are deprecated. To provide additional target
     (``.PHONY`` or not) please use ``CoqMakefile.local``.
 
+.. note::
+
+  Due to limitations with the compilation chain, makefiles generated
+  by ``coq_makefile`` won't correctly work with ``-j N`` (for ``N >
+  1``) for OCaml versions older than 4.07.0.
```

Comment:
This makes it sound like `-j N` won't work correctly for any project if Coq was compiled with an older version of OCaml.  But the version of OCaml should not impact parallelism for .v files.  Is this only for plugin files?  If so, that should be made explicit here.

Reply:
Good point, amended, thanks!